### PR TITLE
List daemonset

### LINKED ISSUE #46: Add a keyword to list daemonsets
Wrapping this kubernetes client funtion will give us the possiblity to list daemonsets.
[list_namespaced_daemon_set](https://github.com/kubernetes-client/python/blob/master/kubernetes/docs/AppsV1Api.md#list_namespaced_daemon_set)

### PR DESCRIPTION
**Pull request by satish-nubolab**
Resolves #46 

Added keyword for list daemon set.

### CODE REVIEW

**Comment by Nilsty** [comment]
@satish-nubolab You are missing an example daemonset for your example test case to work. I'd recommend adding one to the tests-objects-chart (https://github.com/devopsspiral/KubeLibrary/tree/master/test-objects-chart/templates) Similar to the job.yml.
Here you can find an example https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/

In addition I think it would be nice if your test case example could show how to check the status of the deamonset by maybe asserting if `status.desiredNumberScheduled` and `status.numberReady` are equal.

**Comment by m-wcislo** [comment]
Also add entry for your contribution as in https://github.com/devopsspiral/KubeLibrary/blob/master/CHANGELOG.md#in-progress

File: src/KubeLibrary/KubeLibrary.py [hunk 1]
```diff
@@ -569,3 +569,21 @@ def get_ingress_details_in_namespace(self, name, namespace):
         """
         ret = self.extensionsv1beta1.read_namespaced_ingress(name, namespace)
         return ret
+
+    def get_daemonsets_in_namespace(self, namespace, label_selector=""):
+        """Gets a list of available daemonsets.
+        Can be optionally filtered by label. e.g. label_selector=label_key=label_value
+        Returns list of daemonsets.
```
Comment (m-wcislo) [inline src/KubeLibrary/KubeLibrary.py:576]: Please add docs for all the parameters for all new keywords.

 I know it is not the case for all keywords in KubeLIbrary, but we would need that at some point.

File: test-objects-chart/templates/daemonset.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: fluentd-elasticsearch
+  namespace: kubelib-tests
```
Comment (m-wcislo) [inline test-objects-chart/templates/daemonset.yaml:5]: Namespace is defined on chart level, we don't need to hardcode it here.

**Review by m-wcislo — CHANGES_REQUESTED** [review]

File: src/KubeLibrary/KubeLibrary.py (hunk 1 above)
Comment (satish-nubolab) [reply src/KubeLibrary/KubeLibrary.py]: Done.

File: test-objects-chart/templates/daemonset.yaml (hunk 2 above)
Comment (satish-nubolab) [reply test-objects-chart/templates/daemonset.yaml]: Done.

**Review by m-wcislo — APPROVED** [review]

**Comment by m-wcislo** [comment]
@Nilsty any other comments? If not I think we could merge it.

**Review by Nilsty — APPROVED** [review]

**Pull request merged**